### ISSUE TITLE
Transformer model

### ISSUE THREAD

**Issue by Bouncer51** [Priyanshiguptaaa/Flipkart_Product_Categorization#3]
Hi,

Have you not completely implemented the Transformer model and seq2seq+ transformer model proposed in "Don’t Classify, Translate: Multi-Level E-Commerce Product Categorization Via Machine Translation". Maggie Yundi Li, Liling Tan, Stanley Kok. 2018. https://arxiv.org/pdf/1812.05774.pdf .

Is there any missing parts that is yet to be implemented?

TIA

**Comment by Priyanshiguptaaa** [comment]
Hi,

Yes, I've implemented it, I'll add it to the repository shortly!

**Comment by Bouncer51** [comment]
I Am having some issue in executing training code of seq2seq. I got error like: test variable is not defined anywhere, could you help me find what exactly the variable holds?.

There is a line in code with test['category'].apply(lambda x: category_dictionary[x])

TIA

Update: I have got it working by using test_data instead.

I have achieved accuracy of 92% when trained for 5 Epochs:
